dockerspawner/dockerspawner.py

```diff
@@ -216,7 +218,7 @@ def get_container(self):
         self.container_id = ''
     
     @gen.coroutine
-    def start(self, image=None, **extra_create_kwargs):
+    def start(self, image=None):
```

Comment:
I think we had talked about making it both an instance-level config option, and something that could still be passed in here by subclasses (e.g. in the case when you need per-container configuration, like [setting the working directory to be the home directory](https://github.com/compmodels/jupyterhub-deploy/blob/master/roles/jupyterhub_host/files/swarmspawner.py#L49)).